mmv1/third_party/terraform/provider/universe/universe_domain_compute_test.go

```diff
@@ -85,6 +105,23 @@ resource "google_compute_instance_template" "instance_template" {
 `, universeDomain)
 }
 
+func testAccUniverseDomain_basic_disk_image(universeDomain, zone, prefix string) string {
+	return fmt.Sprintf(`
+provider "google" {
+  universe_domain = "%s"
+}
+
+resource "google_compute_disk" "primary" {
+  name  = "async-test-disk"
+  type  = "pd-ssd"
+  zone  = "%s"
+
+  physical_block_size_bytes = 4096
+  image = "projects/%s:debian-cloud/global/images/debian-12"
```

Comment:
Can we inject the full project name here as `prefixedProject` instead of `prefix`? For alternate universe that's `example.com:debian-cloud` instead of `example.com`, and that way we can supply `debian-cloud` for GCP. That'll let us remove the skip as the test will pass in GCP and other universes.

Reply:
Yes, you are right about injecting the full project name here instead of the `prefix`. There's just a little problem for the image name `debian-12` which is not the actual name in different universe. (But looks like this should work in GCP)

Reply:
As discussed offline, can we use `data.google_compute_images` to identify an image appropriate to the environment?